modin/pandas/test/dataframe/test_map_metadata.py

```diff
@@ -1385,13 +1405,22 @@ def comparator(df1, df2):
     for idx, value in enumerate(
         (pandas_df.to_numpy(), np.array([[1]] * 3), np.array([[1, 2, 3], [4, 5, 6]]))
     ):
+        raising_exceptions = None
+        if idx == 0:
+            raising_exceptions = ValueError(
+                "Expected a 1D array, got an array with shape (3, 2)"
+            )
+        elif idx == 2:
+            # Modin's exception message looks better, it's probably fine just leave it as is
```

Comment:
Can you provide an example of the messages from Modin and pandas? Even if our message looks better, why should we not match pandas?

Reply:
I created new issues. Let’s decide it separately?